src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,54 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ResponseError';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  addingMovies: (movie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ addingMovies }) => {
+  const [searchedValue, setSearchedValue] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [errorMassage, setErrorMassage] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchedValue(event.target.value);
+    setErrorMassage('');
+  };
+
+  const preparingMovie = (movieFromServer: MovieData): Movie => {
+    return {
+      title: movieFromServer.Title,
+      description: movieFromServer.Plot,
+      imgUrl: movieFromServer.Poster !== 'N/A'
+        ? movieFromServer.Poster
+        : 'https://via.placeholder.com/360x270.png?text=no%20preview',
+      imdbUrl: `https://www.imdb.com/title/${movieFromServer.imdbID}`,
+      imdbId: movieFromServer.imdbID,
+    };
+  };
+
+  const submitSearch = () => {
+    setIsLoading(true);
+    getMovie(searchedValue).then(res => {
+      if ((res as ResponseError).Error) {
```

Comment:
better to handle an error in .catch

Reply:
How I can handle it in .catch? After 'getMovie()' function I receive Object, and check result's type in 'if'.

Reply:
Often you can have an error from server that'll be in `catch` block. So don't forget to add it.
But now it's ok, sorry for the confusion